jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -265,7 +265,8 @@ static final void initializeClassLoaders() {
 			System.bootLayer = jdk.internal.module.ModuleBootstrap.boot();
 		/*[IF Java10]*/
 		} catch (Exception ex) {
-			ex.printStackTrace();
+			System.out.println(ex);
+			System.out.println("Caused by: " + ex.getCause()); //$NON-NLS-1$
```

Comment:
Actually, can't cause be `null`?

Reply:
Added null check, won't print message for a `null` cause.
Please have another look.